torchsynth/config.py

```diff
@@ -0,0 +1,37 @@
+import warnings
+import torch
+

```

Comment:
Wasn't sure about the name of this module. It is sort of module config type stuff. This function will run when someone imports torchsynth, so they will know right away whether they can produce reproducible results.

Reply:
reproducibility.py?